app_flutter/lib/build_dashboard_page.dart

```diff
@@ -48,37 +139,21 @@ class BuildDashboardPage extends StatelessWidget {
           title: statusTable[_buildState.isTreeBuilding],
           backgroundColor: colorTable[_buildState.isTreeBuilding],
           actions: <Widget>[
-            DropdownButton<String>(
-              value: _buildState.currentBranch,
-              icon: const Icon(
-                Icons.arrow_downward,
-                color: Colors.white,
-              ),
-              iconSize: 24,
-              elevation: 16,
-              style: const TextStyle(color: Colors.white),
-              underline: Container(
-                height: 2,
-                color: Colors.white,
-              ),
-              onChanged: (String branch) {
-                _buildState.updateCurrentBranch(branch);
-              },
-              items: _buildState.branches.map<DropdownMenuItem<String>>(
-                (String value) {
-                  return DropdownMenuItem<String>(
-                    value: value,
-                    child: Text(value),
-                  );
-                },
-              ).toList(),
+            FlatButton(
```

Comment:
Is there a difference between this and `IconButton`?

Reply:
I wasn't aware of IconButton. Fixed.